Source/Workers/decodeDraco.js

```diff
@@ -41,14 +41,9 @@ define([
     }
 
     function decodeQuantizedDracoTypedArray(dracoGeometry, attribute, quantization, vertexArrayLength) {
-        var vertexArray;
-        var attributeData = new draco.DracoInt32Array();
-        if (quantization.octEncoded) {
-            vertexArray = new Int16Array(vertexArrayLength);
-        } else {
-            vertexArray = new Uint16Array(vertexArrayLength);
-        }
-        dracoDecoder.GetAttributeInt32ForAllPoints(dracoGeometry, attribute, attributeData);
+        var vertexArray = new Uint16Array(vertexArrayLength);
+        var attributeData = new draco.DracoUInt16Array();
+        dracoDecoder.GetAttributeUInt16ForAllPoints(dracoGeometry, attribute, attributeData);
```

Comment:
While working on `decodeDracoPointCloud` I realized my suggestion to use `Int16Array` for oct-encoded attributes was incorrect, so I fixed it here as well.